docs/components/connectors/use-connectors/index.md

```diff
@@ -185,3 +185,17 @@ else null
 This is assuming you requested data from a local weather station and received a value that is technically valid for the REST Connector.
 However, you could define that for your business case a humidity value below `0` must be an error that should be checked manually.
 You could automatically send a message to a technician to check the weather station.
+
```

Comment:
Since this is generic, I wonder if this should be earlier in this section, like maybe line 158.

Reply:
Or restructure this section to be more cohesive, putting all similar error handling together, generic and specific. This is something @christinaausley could likely assist with.

Reply:
Ok, I see with @christinaausley . I didn't want to change the structure, just added the missing piece of the puzzle.

Reply:
I agree here on keeping all things error handling together -- let me know if I can assist further or if you're looking for additional suggestions.

Reply:
Slacking you on this in the case we want to discuss further @pierre-yves-monnet 👍